source/features/show-associated-branch-prs-on-fork.tsx

```diff
@@ -84,8 +102,22 @@ async function addLink(branchCompareLink: Element): Promise<void> {
 	}
 }
 
+async function addLinkNew(prAnchorElement: HTMLAnchorElement): Promise<void> {
+	const prs = await pullRequestsAssociatedWithBranch.get();
+	const tableRow = prAnchorElement.closest('.TableRow') as HTMLTableRowElement;
+	const branchName = (tableRow.firstChild! as HTMLTableCellElement).querySelector('div[title]')!.getAttribute('title')!;
+	const prInfo = prs[branchName];
```

Comment:
You probably want to select the branch name in `observe` and then get only the `prCell` from there, so that we limit the amount of DOM traversing.